### DISCUSSION TITLE
Override brew repo owner

### DISCUSSION THREAD

**Discussion by jvalkeal** [jreleaser/jreleaser#1566] [General]
In `spring-cli` we wanted to move brew tap repo from `spring-projects` org into `spring-cli-projects` org while keeping `spring-cli` in `spring-projects`. What was nice about not defining `distributions.<app>.brew.repository.owner`, owner then defaulted wherever jreleaser was run in gh actions. This allowed to fully test your release process in a user's fork.

I have a feeling that answer is no but is there a generic way to define some of these jrerealser config values so that those would not be hardcoded to target specific repository owners or names? Looks like `brew.repository.owner` can't be resolved from i.e. from environment variables.

**Comment by aalmiray** [comment]
That's right, not all DSL elements may be overridden using sysprops/envvars. Initially this was only possible for sensitive data such as tokens and secrets. However, we have expanded this behavior to more properties, even those that are not secrets. 

It may be possible to support most if not all properties (indexed elements may be tricky) in a compatible way. Otherwise we'd have to punt it to `v2`.
